src/app/ngrx/reducers/podcast.reducer.ts

```diff
@@ -0,0 +1,53 @@
+import { Action } from '@ngrx/store';
+import ActionTypes from '../actions/action.types';
+import { PodcastModel } from '../model';
+
+const initialState = [];
+
+export function PodcastReducer(state: PodcastModel[] = initialState, action: Action) {
+  let podcastIdx: number, epIdx: number, podcast: PodcastModel, newState: PodcastModel[];
+  switch (action.type) {
+    case ActionTypes.CMS_PODCAST_FEED:
+      podcastIdx = state.findIndex(p => p.seriesId === action.payload.podcast.seriesId);
+      if (podcastIdx > -1) {
+        podcast = Object.assign({}, state[podcastIdx], {
+          doc: action.payload.podcast.doc,
+          seriesId: action.payload.podcast.seriesId,
+          title: action.payload.podcast.title,
+          feederUrl: action.payload.podcast.feederUrl,
+          feederId: action.payload.podcast.feederId
```

Comment:
It will eventually be possible that episodeIds could exist in `state[podcast]`, so it's retaining prior state if the entry exists, but it's not going to get any updates to episodeIds from getting the series + series podcast distribution. episodeIds should only come from getting the story + episode distribution.